pkg/nidhogg/handler.go

```
@@ -61,8 +61,22 @@ type Handler struct {
 
 // HandlerConfig contains the options for Nidhogg
 type HandlerConfig struct {
-	Daemonsets   []Daemonset       `json:"daemonsets" yaml:"daemonsets"`
-	NodeSelector map[string]string `json:"nodeSelector" yaml:"nodeSelector"`
+	Daemonsets   []Daemonset `json:"daemonsets" yaml:"daemonsets"`
+	NodeSelector []string    `json:"nodeSelector" yaml:"nodeSelector"`
+	Selector     labels.Selector
+}
+
+func (hc *HandlerConfig) BuildSelectors() {
+	print("test")
+	hc.Selector = labels.Everything()
+	for _, rawSelector := range hc.NodeSelector {
+		if selector, err := labels.Parse(rawSelector); err != nil {
+			panic(err)
```

this should return an error instead if panicking